test/ValidateCheckWebauthnTest.php

```diff
@@ -100,8 +100,6 @@ public function testTriggerWebAuthn()
         $this->assertEquals("Please confirm with your WebAuthn token (Yubico U2F EE Serial 61730834)", $response->multiChallenge[0]->message);
         $this->assertEquals("WAN00025CE7", $response->multiChallenge[0]->serial);
         $this->assertEquals("webauthn", $response->multiChallenge[0]->type);
-        $this->assertArrayHasKey("img", $response->multiChallenge[0]->attributes);
```

Comment:
is the new field `image` covered with an assert?

Reply:
[f3ed482](https://github.com/privacyidea/php-client/pull/31/commits/f3ed48283680e1a6b632f3e8a886218a1325474a)